### PR TITLE
{lang}[intel/2016b] Python v2.7.12

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
depends on #3270, #3276

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 7 out of 9 (9 easyconfigs in this PR)
node2126.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/efef96af557bd16fdd3610c6321237b3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 7 out of 9 (9 easyconfigs in this PR)
node2420.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/b8e94f7303496c54d63945f3ad097b79 for a full test report.

File: easybuild/easyconfigs/p/Python/Python-2.7.12-intel-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+sources = [SOURCE_TGZ]
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.8'),
+    ('libreadline', '6.3'),
+    ('ncurses', '6.0'),
+    ('SQLite', '3.13.0'),
+    ('Tk', '8.6.5', '-no-X11'),
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/Python/Python-2.7.12-intel-2016b.eb]: this is going to cause version hell if something needs Tk with X?

Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-2.7.12-intel-2016b.eb]: this hasn't been an issue up until now, why should it suddenly become a problem?

anyway, we need to work towards a single Tk that does not rely on system X11 deps anyway, so we should probably just drop the versionsuffix here, and then later enhance the Tk easyconfig to include an X11 bundle (once we have that in place)...

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 8 out of 9 (9 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/9f87a2eac04ccfa3b71655e2a48b1e8c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 8 out of 9 (9 easyconfigs in this PR)
node2611.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/07341b4e38a87ed3ed3764ae9d5af8a6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 10 out of 10 (10 easyconfigs in this PR)
node2022.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/804e3baa86af37ec1583faec3096ea5a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 10 out of 10 (10 easyconfigs in this PR)
node2470.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/2854ce5c61a7f7f1fc248ab4cf7ddac3 for a full test report.

**Comment by boegel** [comment]
'just' a version bump, versions of extensions and dependencies were also bumped

**Pull request merged**